src/operations/bulk_write.ts

```diff
@@ -45,7 +45,7 @@ export class BulkWriteOperation extends OperationBase<BulkWriteOptions, BulkWrit
         : coll.initializeUnorderedBulkOp(options);
 
     // Do we have a collation
-    let collation = false;
+    const collation = false;
```

Comment:
collation is never changed, there's an if statement about 20 lines down that will never run, just wanted to get a second pair of eyes that this is correct and its okay to delete?